src/pylhe/__init__.py

```diff
@@ -300,11 +305,24 @@ def read_lhe(filepath):
         return
 
 
+def _get_index_to_id_map(init):
+    """
+    Convert index to id.
```

Comment:
Can you improve the docstring to state the cases where this mapping is relevant (I see from the caller below) and state what the argument is and what variables are returned? Not everyone is expert in POWHEG ;-).